### PR TITLE
update footer links

### PR DESCRIPTION
**Pull request by Dhaulagiri**
### :pushpin: Summary

Updates footer links per my discussion with Legal

### :link: External links

Jira ticket: [HDS-1200](https://hashicorp.atlassian.net/browse/HDS-1200)

### CODE REVIEW

**Review by jorytindall — APPROVED** [review]
Links look fine, I'm sure this was done intentionally (more curious than anything and not directly related to this PR) but the DOM order doesn't match the visual order.

File: website/app/components/doc/page/footer.hbs [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
-    <a href="#">Terms of Use</a>
-    <a href="#">Privacy</a>
+    <LinkTo @route="show" @model="about/accessibility-statement">Accessibility</LinkTo>
+    <a href="https://hashicorp.com/careers" target="_blank" rel="noopener noreferrer">Careers</a>
+    <a href="https://www.hashicorp.com/terms-of-service" target="_blank" rel="noopener noreferrer">Terms of Use</a>
+    <a href="https://www.hashicorp.com/security" target="_blank" rel="noopener noreferrer">Security</a>
+    <a href="https://www.hashicorp.com/privacy" target="_blank" rel="noopener noreferrer">Privacy</a>
     <span role="presentation">–</span>
     <LinkTo @route="testing">Testing</LinkTo>
     <button type="button" class="doc-page-footer__🌈" {{on "click" this.toggleLayoutColors}}>🌈</button>
```
Comment (MelSumner) [inline website/app/components/doc/page/footer.hbs:10]:
```suggestion
    <button type="button" class="doc-page-footer__🌈" {{on "click" this.toggleLayoutColors}} aria-label="toggle layout colors">🌈</button>
```

**Review by MelSumner — APPROVED** [review]
Approve the content but these need to be inside of a list (and preferably a nav with a unique accname); that can be done in a separate PR though.

File: website/app/components/doc/page/footer.hbs [hunk 2]
```diff
@@ -1,15 +1,21 @@
...
   </div>
-  <a class="doc-page-footer__logo" href="https://www.hashicorp.com/" target="_blank" rel="noopener noreferrer" aria-label="hashicorp dot com">
+  <a
+    class="doc-page-footer__logo"
+    href="https://www.hashicorp.com/"
+    target="_blank"
+    rel="noopener noreferrer"
+    aria-label="hashicorp dot com"
```
Comment (KristinLBradley) [inline website/app/components/doc/page/footer.hbs:17]: @MelSumner Does this make sense as the aria-label?

File: website/app/components/doc/page/footer.hbs (hunk 1 above)
Comment (Dhaulagiri) [reply website/app/components/doc/page/footer.hbs:10]: this will be removed before launch

**Comment by Dhaulagiri** [comment]
I'll merge these changes in isolation and we can re-organize the DOM in a followup

**Pull request merged**